src/imitation/algorithms/preference_comparisons.py

```diff
@@ -1202,7 +1203,7 @@ def train(
         self,
         total_timesteps: int,
         total_comparisons: int,
-        callback: Optional[Callable[[int], None]] = None,
+        callback: Optional[Callable[[int, int], None]] = None,
```

Comment:
Probably should add in the docstring what the callback type signature represents.